platform/authorized_api_guide.md

```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
+is to perform basic operations on an app. We will:
+* create an app
+* get created app
+* update its some fields
+* in the end, we will remove given app
+
+#### Generate your access token
+
+Request example:
 
 ```
-curl 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
+POST /auth/token
+```
+
+```
+curl -X POST 'https://<domain>/auth/token' -H "Content-Type: application/json" --data '{
     "grant_type": "client_credentials",
     "client_id": "your_generated_client_id",
     "client_secret": "your_generated_client_secret"
 }' 
 ```
 
-The example response is: 
+Response example:
 ```
-{"token_type":"Bearer","expires_in":86400,"access_token":"your_access_token"}
+{
+    "token_type":"Bearer",
+    "expires_in":86400,
+    "access_token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJwcG1zIiwiYXVkIjoiaHR0cHM6XC9cL3Rlc3RpbmcucGl3aWsucHJvXC9zZXR5LCJzdWIiOiJkNmNkZGMxMS1iZDA1L0aW5ncyIsImlhdCI6MTUzNzI3MDQ1OSwiZXhwIjoxNTM3MzU2ODUTRhYmUtYWIyZC02YjlhNjIxZmU0ZDciLCJvcmciOiJkZWZhdWx0In0.Nec2mYFRv6manzXjq0sHQxINZvu-fbDYT8AedVHBKYvu1F9hYKaFReY8rNgfsMANw2OX8-IKpTrQb1DyRkG4nxpIEbob528_lPd7roho5mtKlE8sfS9WZE1piYOwaNDySDEUwUowgj2xBiJqSODjxBI6qVhLkynGEEeNBVh-lrUmlcjpYqUc3saHvX72L-rqbIHa_1dzGarR-dcPyns-RpKjZEILzUSYOHdM09KDti-xsG-nbKHGdP8fVEEJPyupnAfJPOLHQg_j1c5IvJSvTKVF3j4_zo6Zw5g8YkaheT9Iwph5BGHFRneXatcmbwKI8JzSDFi6CinzI-okYKRPbg"
+}
 ```
 
-Piwik PRO API tokens use [jwt](https://jwt.io/) format.
+Field `access_token` contains your token which then will be used for all API calls.
+Once you generated an access token, you can use it during its lifetime (24 hours by default)
+
+#### Create an app
 
-Now, you can use obtained `access_token` for communication with Piwik PRO API.
-Field `expires_in` stands for time (in seconds) for token TTL.
-As token is a Bearer type, it must be **included in all API calls** within header.
+Request example:
 
 ```
-Authorization: Bearer your-token-here
+POST /api/apps/v2
 ```
-  
-## Deleting API Credentials
 
-Once you want to revoke the possibility of generating API token using given `CLIENT ID` and `CLIENT SECRET`,
-go to to `My profile` and click `Delete` button on selected API credentials.
+```
+curl -X POST 'https://<domain>/api/apps/v2' -H "Authorization: Bearer <your_access_token>" -H "Content-Type: application/vnd.api+json" --data '{
+  "data": {
+    "attributes": {
+      "timezone": "UTC",
+      "name": "AppName",
+      "urls": [
+        "http://example.com"
+      ],
+      "currency": "USD"
+    },
+    "type": "ppms/app"
+  }
+}'
+
+```
+
+Note, that you have to replace: 
+* `<domain>` with your PPMS instance URL,
+* `<your_access_token>` with your generated access token 
+
+Response example:
+```
+{
+   "data":{
+      "type":"ppms\/app",
+      "id":"b30e538d-4b05-4a75-ae25-7eb565901f38",
+      "attributes":{
+         "name":"AppName",
+         "addedAt":"2018-09-13T12:16:30+00:00",
+         "urls":[
+            "http:\/\/example.com"
```

Comment:
Is this correct? I don't think JSON escapes slashes, but it shouldn't affect build docs. It's just harder to read in the source.

Reply:
yes it is semantically correct, but as you pointed it's not needed